Src/WitsmlExplorer.Frontend/components/ContentViews/CurveValuesView.tsx

```diff
@@ -29,6 +30,7 @@ const DEPTH_INDEX_START_OFFSET = 20; // offset before log end index that defines
 const TIME_INDEX_OFFSET = 30536000; // offset from current end index that should ensure that any new data is captured (in seconds).
 const DEPTH_INDEX_OFFSET = 1000000; // offset from current end index that should ensure that any new data is captured.
 const DEFAULT_REFRESH_DELAY = 5.0; // seconds
+const AUTO_REFRESH_TIMEOUT = 0.2; // minutes
```

Comment:
Remember to update this value back to 5.0.

Reply:
Good catch, fixed 😅